### ISSUE TITLE
Uploaded photos not displaying on data submissions for admin users

### ISSUE THREAD

**Issue by StephenAbbott** [okfn/crowdsourced-surveys#9]
This report was submitted with a photo but the photo is not showing up in the review interface https://crowdsurvey.herokuapp.com/posts/31

**Comment by StephenAbbott** [comment]
Same issue experienced with https://crowdsurvey.herokuapp.com/posts/41 + https://crowdsurvey.herokuapp.com/posts/71

**Comment by nightsh** [comment]
I tracked this down to an undocumented configuration variable that was hardcoded in an options object within the app. Modifying that from an initial 1MB to 10MB and restarting the API container turned out to fix the issue:

```
$ heroku run -a crowdsurvey-api sed -i s/1048576/10485760/ /app/application/config/media.php
$ heroku restart -a crowdsurvey-api
```

As far as other deployment methods go, the same setting should be applied (while also paying attention to web server maximum allowed payload size).

Closing as fixed in demo site and fixable everywhere in the future.
